### PR TITLE
Add a recipe for HTML forms and a recipe for JS forms

### PR DESCRIPTION
**Pull request by Jutanium**
To be merged after #2550

#### What kind of changes does this PR include?

- New or updated content

#### Description

These are the two forms recipes from #2550, in their own PR. The first ex

### CODE REVIEW

File: src/content/docs/en/recipes/build-forms-api.mdx [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        ```
+      </Fragment>
+
+    </UIFrameworkTabs>
+
+  2. Create the `post` API endpoint that will receive the form data. Use `request.formData()` to process it. Be sure to validate the form values before you use them. 
+  
+        This example sends a JSON object with a message back to the client.
+
+      ```ts title="src/api/feedback.ts" "request.formData()" "post"
```
Comment (Yan-Thomas) [inline src/content/docs/en/recipes/build-forms-api.mdx:140]: The API route should be inside pages, right?
```suggestion
      ```ts title="src/pages/api/feedback.ts" "request.formData()" "post"
```

File: src/content/docs/en/recipes/build-forms-api.mdx [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+        if (!name || !email || !message) {
+          return new Response(
+            JSON.stringify({
+              message: "Missing required fields",
+            }),
+            { status: 400 }
+          );
+        }
+        // Do something with the data, then return a success response
+        return new Response({ message: "Submitted!"}, { status: 200 });
```
Comment (kevinzunigacuellar) [inline src/content/docs/en/recipes/build-forms-api.mdx:158]: the object response needs to be a string 😉
```suggestion
        return new Response(
          JSON.stringify({
            message: "Success!"
          }),
          { status: 200 }
        );
```

The response needs to be stringified

**Review by kevinzunigacuellar — COMMENTED** [review]
🧑‍🍳

File: src/content/docs/en/recipes/build-forms-api.mdx (hunk 2 above)
Comment (Jutanium) [reply src/content/docs/en/recipes/build-forms-api.mdx:158]: Thanks!

**Pull request merged**